test/spec/features/popup-menu/ReplaceMenuProviderSpec.js

```diff
@@ -1991,3 +1879,31 @@ describe('features/popup-menu - replace menu provider', function() {
   });
 
 });
+
+
+
+// helper /////
+function queryEntry(id) {
+  return domQuery('.djs-popup [data-id="' + id + '"]');
```

Comment:
I believe we must scope this one to the current diagram-js instance, right? This is what we accomplished by passing the popupMenu in the past. Otherwise we assert that the entry from the first matching test run is there, not the current one.

Reply:
Yes and no. 

**Yes**, because at the moment we query the whole document which surely slows tests down. 

**No**, because there cannot be a second popup menu open since the old diagram is [always destroyed before a next one is created](https://github.com/bpmn-io/bpmn-js/blob/33156e43be92b57bdef0f3b7e965ce36b105fe1f/test/helper/index.js#L117). This is why the tests didn't break.

Anyway, I pushed a scoped container queries on top because it's surely not a good practice to query outside of the test container.